### PR TITLE
fix "No module named xxx" error after built target executable file by…

### PR DESCRIPTION
**Pull request by kerneltravel**
fix "No module named xxx" error after built target executable file by Pyinstaller and run. the related issue ID is #10398, #10340, #9951, #9723 #9694. according to the suggestion from issue #9408.

### CODE REVIEW

File: ppocr/data/imaug/fce_aug.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def generate_crop_target(self, image, all_polys, pad_h, pad_w):
         for polygon in all_polys:
             rect = cv2.minAreaRect(polygon.astype(np.int32).reshape(-1, 2))
             box = cv2.boxPoints(rect)
-            box = np.int0(box)
+            box = np.int64(box)
```
Comment (onecatcn) [inline ppocr/data/imaug/fce_aug.py:211]: please explain the reason for making this change. it seems irrelevant to the purpose of the PR.

File: tools/infer/utility.py [hunk 2]
```diff
@@ -220,7 +220,7 @@ def create_predictor(args, mode, logger):
                 logger.warning(
                     "GPU is not found in current device by nvidia-smi. Please check your device or ignore it if run on jetson."
                 )
-            config.enable_use_gpu(args.gpu_mem, args.gpu_id)
+            config.enable_use_gpu(args.gpu_mem, 0)
```
Comment (onecatcn) [inline tools/infer/utility.py:223]: please explain the reason for making this change. it seems irrelevant to the purpose of the PR.

File: tools/infer/utility.py [hunk 3]
```diff
@@ -246,7 +246,7 @@ def create_predictor(args, mode, logger):
                     logger.info("Please keep your paddlepaddle-gpu >= 2.3.0!")
 
         elif args.use_npu:
-            config.enable_custom_device("npu")
+            config.enable_npu()
```
Comment (onecatcn) [inline tools/infer/utility.py:249]: please explain the reason for making this change. it seems irrelevant to the purpose of the PR.

**Review by onecatcn — COMMENTED** [review]
I have comments on 3 places, where the code change seems irrelevant to the purpose of the PR.

File: tools/infer/utility.py (hunk 2 above)
Comment (kerneltravel) [reply tools/infer/utility.py:223]: sorry, this change is wrong. Now have re-PR: https://github.com/PaddlePaddle/PaddleOCR/pull/10421

File: tools/infer/utility.py (hunk 3 above)
Comment (kerneltravel) [reply tools/infer/utility.py:249]: sorry, this change is wrong. Now have re-PR: https://github.com/PaddlePaddle/PaddleOCR/pull/10421
